pkg/config/config_template.yaml

```diff
@@ -2753,3 +2753,24 @@ api_key:
   # stop_timeout: 5.0
 
 {{end -}}
+{{- if .OpenTelemetry }}
+
+###################################
+## OpenTelemetry Configuration   ##
+###################################
+
+## @param otlp - custom object - optional
+## This section configures the OpenTelemetry OTLP receiver.
+#
+# otlp:
```

Comment:
Since we're already in the configuration file `_config` is redundant, but I think consistency is more important. It will make the experience more intuitive for the users.

Reply:
I agree fully, consistency is better despite the redundancy.

Reply:
I'd argue that we have `_config` on our own products (`apm_config`, `logs_config`, etc), and `otlp` is not. Also, having `_config` on each and every top-level key is a bit much. Care to re-consider? Another argument is that this was `otlp` when it was in the `experimental` section too. To upgrade, the user only needs to remove the top level section.

Reply:
@hush-hush @truthbk @bkabrda see Gabriel's comment above. I don't have strong feelings so I think I am going to abstain here (if you think this should be handled differently please speak up)

Reply:
Done.